### PR TITLE
LPS-126378

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/path/DataEngineBuilder.path [hunk 1]
```diff
@@ -58,6 +58,11 @@
 	<td>//button[(@class='dropdown-toggle btn btn-monospaced btn-unstyled')]</td>
 	<td></td>
 </tr>
+<tr>
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/path/DataEngineBuilder.path:61]: Use existing locator

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case duplicates with LocalizedNestedFieldsStructure#TranslationDoesNotChangeAnother  
See https://github.com/liferay-qa/liferay-portal/pull/174

**Comment by emanuelfisilva** [comment]
Hey @Tim-Cao , thanks for your review. We will choose to keep this test as per now, and work on refactor it later as per defined strategy.

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3706
View total diff https://github.com/liferay-echo/liferay-portal/pull/3706/commits/783428b9ad8de606d076e68148894932be16884d

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**